src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -237,6 +289,35 @@ private string ClearHexString(string hexString)
             }
         }
 
+        /// <summary>
+        /// Converts a BigInteger to hex string to UTF8 string
+        /// input:  860833102
+        /// output: NEO3
+        /// </summary>
+        /// <param name="strParam">
+        /// String that represents the number to be converted
+        /// </param>
+        /// <returns>
+        /// Returns null when the string does not represent a big integer value or when
+        /// it is not possible to parse the big integer value to UTF8 string value; otherwise,
+        /// returns the string that represents the converted UTF8 string value
+        /// </returns>
+        private string? NumberToHexToString(string input)
+        {
+            if (!new Regex("^\\d+$").IsMatch(input) || input.StartsWith('0'))
```

Comment:
this way to slow. use `BigInteger.TryParse`

Reply:
Already modified